static/app/components/versionHoverCard.tsx

```diff
@@ -142,68 +141,52 @@ class VersionHoverCard extends Component<Props, State> {
     };
   }
 
-  render() {
-    const {
-      deploysLoading,
-      deploysError,
-      release,
-      releaseLoading,
-      releaseError,
-      repositories,
-      repositoriesLoading,
-      repositoriesError,
-    } = this.props;
-    let header: React.ReactNode = null;
-    let body: React.ReactNode = null;
-
-    const loading = !!(deploysLoading || releaseLoading || repositoriesLoading);
-    const error = deploysError ?? releaseError ?? repositoriesError;
-    const hasRepos = repositories && repositories.length > 0;
-
-    if (loading) {
-      body = <LoadingIndicator mini />;
-    } else if (error) {
-      body = <LoadingError />;
-    } else {
-      const renderObj: {[key: string]: React.ReactNode} =
-        hasRepos && release ? this.getBody() : this.getRepoLink();
-      header = renderObj.header;
-      body = renderObj.body;
-    }
-
-    return (
-      <Hovercard {...this.props} header={header} body={body}>
-        {this.props.children}
-      </Hovercard>
-    );
+  let header: React.ReactNode = null;
+  let body: React.ReactNode = null;
+
+  const loading = !!(deploysLoading || releaseLoading || repositoriesLoading);
+  const error = deploysError ?? releaseError ?? repositoriesError;
+  const hasRepos = repositories && repositories.length > 0;
+
+  if (loading) {
+    body = <LoadingIndicator mini />;
+  } else if (error) {
+    body = <LoadingError />;
+  } else {
+    const renderObj: {[key: string]: React.ReactNode} =
+      hasRepos && release ? getBody() : getRepoLink();
+    header = renderObj.header;
+    body = renderObj.body;
   }
+
+  return (
+    <Hovercard {...hovercardProps} header={header} body={body}>
+      {children}
+    </Hovercard>
+  );
 }
 
 interface VersionHoverHeaderProps {
   releaseVersion: string;
 }
 
-export class VersionHoverHeader extends Component<VersionHoverHeaderProps> {
-  render() {
-    return (
-      <HeaderWrapper>
-        {t('Release')}
-        <VersionWrapper>
-          <StyledVersion version={this.props.releaseVersion} truncate anchor={false} />
-
-          <CopyToClipboardButton
-            borderless
-            iconSize="xs"
-            size="zero"
-            text={this.props.releaseVersion}
-          />
-        </VersionWrapper>
-      </HeaderWrapper>
-    );
-  }
+function VersionHoverHeader({releaseVersion}: VersionHoverHeaderProps) {
+  return (
+    <HeaderWrapper>
+      {t('Release')}
+      <VersionWrapper>
+        <StyledVersion version={releaseVersion} truncate anchor={false} />
+        <CopyToClipboardButton
+          borderless
+          iconSize="xs"
+          size="zero"
+          text={releaseVersion}
+        />
+      </VersionWrapper>
+    </HeaderWrapper>
+  );
 }
 
-export {VersionHoverCard};
 export default withApi(withRelease(withRepositories(VersionHoverCard)));
```

Comment:
Do all these HOCs have hooks we can use now?

Reply:
not really